trackpy/linking.py

```diff
@@ -546,6 +547,10 @@ def link_df(features, search_range, memory=0,
     if hash_size is None:
         MARGIN = 1  # avoid OutOfHashException
         hash_size = features[pos_columns].max() + MARGIN
+    if features.is_copy is not None and not copy_features:
+        warn('The features DataFrame is a view, so it is not writeable. '
+             'The results will be outputted to a copy.')
```

Comment:
At the risk of being too verbose, it might be nice if this warning offered a more permanent solution: 'Use the "copy_features" argument to remove this warning.'